test/test_jra55.jl

```diff
@@ -129,5 +129,22 @@ using ClimaOcean.OceanSeaIceModels: PrescribedAtmosphere
         @test rivers_times != pressure_times
         @test length(rivers_times) != length(pressure_times)
         @test rivers_times[2] - rivers_times[1] == 86400
+
+        @info "Testing multi year JRA55 data on $A..."
+        dataset = JRA55MultipleYears()
+        dates = ClimaOcean.DataWrangling.all_dates(JRA55MultipleYears(), :temperature)
+        
+        # These dates correspond to a metadata that crosses between year 1958 and 1959.
+        # Therefore, this will download two files for the two years and concatenate them
+        # when reading the data.
+        start_date = dates[2800]
+        end_date   = dates[3600]
+        backend = JRA55NetCDFBackend(10)
+        Ta = JRA55FieldTimeSeries(:temperature; dataset=JRA55MultipleYears(), start_date, end_date, backend)
```

Comment:
when we create a `JRA55FieldTimeSeries` with `JRA55MultipleYears()` the time indexing seems to be `Cyclical()`? Does this mean that if we run the simulation longer than the range of the dataset then it will loop from the start? Should we keep this as default?

Reply:
I like that as a default

Reply:
Minor: Perhaps just spit out an info when this happens?
Imagine somebody starts a simulation in 2020 and they run for what they think is 4 years but during that time they hit the end of the dataset and they go back to 1958 without them knowing

Reply:
Since we are at it, I never understood what the eg 10 in `JRA55NetCDFBackend(10)` does. Is it how many time-slices of the dataset are loaded into memory?

Reply:
yep, it's the number of indices stored in memory, the rest is on disk

Reply:
Thanks!